vendor/vendor.json

```diff
@@ -182,6 +194,12 @@
 			"revisionTime": "2016-08-11T16:24:59Z"
 		},
 		{
+			"checksumSHA1": "qS3eZZfhdDM6haTZp7OeyhMDlV8=",
+			"path": "github.com/aws/aws-sdk-go/service/route53domains",
+			"revision": "f80e7d0182a463dff0c0da6bbed57f21369d4346",
```

Comment:
I'd be ok updating everything that needs it. We can run the integration tests before merging to make sure everything else still works.